### DISCUSSION TITLE
GSoC 2024

### DISCUSSION THREAD

**Discussion by gunjjoshi** [meshery/meshery#10209] [General] [answered]
Are we going for GSoC 2024 ? If yes, where can I find guidelines for our organization ? 
Also, any communication channel link if available ?

**Comment by Yashsharma1911** [comment, marked as answer]
Hey @gunjjoshi thanks for engaging with us here, can you please post the same question on [discussion forum](http://meshery.io/community#discussion-forums) so that other community members can interact with you and can answer your questions

**Reply by gunjjoshi** [reply]
Sure, will definitely do that.

**Reply by Yashsharma1911** [reply]
Thankyou
